src/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <section id="stripped_db_provider" translate="label" type="text" sortOrder="30" showInDefault="1" showInWebsite="1" showInStore="1">
+            <tab>wearejh</tab>
+            <label>DB Provider</label>
+            <resource>Jh_StrippedDbProvider::config</resource>
+            <group id="general" translate="label"  type="text" sortOrder="0" showInDefault="1" showInStore="0" showInWebsite="0">
+                <label>General</label>
+                <field id="cron_expr" translate="label" type="text" sortOrder="10" showInDefault="1" showInWebsite="0" showInStore="0">
+                    <label>Cron Expression</label>
+                </field>
+            </group>
+            <group id="storage" translate="label"  type="text" sortOrder="10" showInDefault="1" showInStore="0" showInWebsite="0">
+                <label>Storage Details</label>
+                <field id="access_key_id" translate="label" type="obscure" sortOrder="10" showInDefault="1" showInWebsite="0" showInStore="0">
+                    <label>Access Key Id</label>
+                    <backend_model>Magento\Config\Model\Config\Backend\Encrypted</backend_model>
+                </field>
+                <field id="secret_access_key" translate="label" type="obscure" sortOrder="20" showInDefault="1" showInWebsite="0" showInStore="0">
+                    <label>Secret Access Key</label>
+                    <backend_model>Magento\Config\Model\Config\Backend\Encrypted</backend_model>
+                </field>
```

Comment:
are all settings now accessible in admin?

Reply:
Cron is good to have - please add all fields into admin config - if we have one, it is logical to have them all. Acl would be nice as well